### PR TITLE
chore(deps): Bump polkadot-js/api from 1.31.2 to 1.34.1

### PR DESCRIPTION
**Pull request by emostov**
### overview

In [polkadot-js/api v1.34.1](https://github.com/polkadot-js/api/releases/tag/v1.34.1) there is a major breaking change and an update to accommodate metadata v12.

### breaking change recap

The breaking change is that metadata creation now does not default to returning decorated metadata. Instead the user must explicitly create the metadata, set in on the registry, and then use that registry to create to create metadata. 

In the workflow for `createDecorated` we create `Metadata` and then use that to create `Decorated`. However, anytime `Decorated` is used, the user must be careful to `registry.setMetadata` first. The only place Decorated is used is within `createMethod` so this isn't a big issue, but to ease future maintenance it may be worth discussing moving `registry.setMetadata` to within `createDecorated`.

In sum, after evaluation I have concluded the breaking change does not impact `txwrapper`. In an abundance of caution I have also tested this branch with several different transactions against and a dev node and had zero issues. It may be worth having someone else pull down this branch and do the same.

### update to accommodate metadata v12

From what I understand, due to this [substrate pr](https://github.com/paritytech/substrate/pull/6969), polkadot-js is bumping the metadata to v12. It is not totally clear to me if this interacts with any of txwraper's use cases, but under the hood polkadot-js is already using this new metadata version and since I tested some transactions I think we are ok.

### other

I needed to add an explicit import for `fetch` to the utils file in the example directory since it was complaining. Not sure why this was not an issue before, but it is fixed now. Likely some dependency got auto updated and since there are no tests for the example folder it did not get caught.

### CODE REVIEW

File: examples/util.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@
  */ /** */
 
 import { TRANSACTION_VERSION } from '@polkadot/types/extrinsic/v4/Extrinsic';
+import fetch from 'node-fetch';
```
Comment (joepetrowski) [inline examples/util.ts:10]: Why does this get imported in this PR but not used?

**Review by joepetrowski — APPROVED** [review]

File: examples/util.ts (hunk 1 above)
Comment (emostov) [reply examples/util.ts:10]: It is used on line 22. I can't figure out exactly why it did not need to be imported before, but while trying out the example it was failing after bumping deps. In modern web browsers fetch is automatically included in the global scope while node does not include it.
